src/main/java/com/endava/endabank/exceptions/customexceptions/ResourceNotFoundException.java

```diff
@@ -1,4 +1,4 @@
-package com.endava.endabank.exceptions.customExceptions;
+package com.endava.endabank.exceptions.customexceptions;
```

Comment:
Hi,

To avoid having a package name without camelcase rule and seeing that customexceptions is a subpackage of exceptions, it is not necessary to name it customexceptions, rather than you can name it only custom